apps/deploy-web/src/utils/deploymentData/v1beta3.ts

```diff
@@ -22,6 +25,51 @@ const getDenomFromSdl = (groups: any[]): string => {
   return denoms[0];
 };
 
+export function appendTrialAttribute(yamlStr: string) {
+  const sdl = getSdl(yamlStr, "beta3", networkStore.selectedNetworkId);
+
+  Object.keys(sdl.data.profiles.placement).forEach(placement => {
+    if (!sdl.data.profiles.placement[placement].attributes) {
+      sdl.data.profiles.placement[placement].attributes = [];
+    }
+
+    if (sdl.data.profiles.placement[placement].attributes.find(attr => attr.key === trialAttribute)) {
```

Comment:
real hard to understand why this is a break condition. Storing it into into a verbose var would help.
```suggestion
    const isBreakCondition = sdl.data.profiles.placement[placement].attributes.find(attr => attr.key === trialAttribute);
    
    if (isBreakCondition) {
```

Reply:
after second look I got it's missing trial attr. However this is still the reason to approach this like so :)

Reply:
Sure